### ISSUE TITLE
Octree geometry seems to not work

### ISSUE THREAD

**Issue by pepsuncho** [nrc-cnrc/EGSnrc#614]
In some of my old input files (<2015) i have used the Octree geometry, which worked pretty well.
Now i wanted to use it again (with exactly the same input file), but it was not working (using the 2020 release of EGSnrc).
After that i have tried also the example geometries (octree-car/octree-sphere) and they seem also not to work. I get the following for the octree-geometry (octree-sphere):

>  type = EGS_Octree
>  name = my_octree
>  number of regions = 1
>  bounding box minimum     = -1 -1 -1
>  bounding box maximum     = 1 1 1
>  bounding box resolution  = 32 32 32
>  octree leaf size         = 0.0625 0.0625 0.0625
>  octree cells (no medium) = 1
>  octree cells (medium)    = 0
>  octree average cell size = -nan
>  octree cell savings      = -nan%

Am i missing something, or the functionality of the octree-geometry is broken?

**Comment by rtownson** [comment]
I've gone back to the start of the git version tracking (2015) and it seems the issue has existed since then. I guess we don't use the octree often! So you aren't missing anything - it does seem to be broken. I'll look into it...

**Comment by ftessier** [comment]
Indeed, I never _completely_ finished the octree geometry (in terms of proper testing and polishing up the input interface for it. I will also look at it to see what has happened since the original implementation.
